docs/source/tutorial/introduction.md

```diff
@@ -1,74 +1,80 @@
+The app includes the following views:
 
-We want this to model a real world Apollo app as much as possible, so we're covering essential topics like authentication, pagination, state management, testing, and deployment.
+* A login page
+* A list of upcoming launches
+* A detail view for an individual launch
+* A user profile page
+* A cart
 
-## Prerequisites
+To populate these views, our app's data graph will connect to two data sources: 
+a REST API and a SQLite database. (Don't worry, you don't need to be familiar with
+either of those technologies to complete the tutorial.)
 
-The tutorial assumes that you're comfortable with JavaScript/ES6, you've fetched data from an API before, and you have basic familiarity with React. If you need to brush up on your React skills, we recommend going through the [official tutorial](https://reactjs.org/tutorial/tutorial.html). Building your frontend with React is not a requirement for using Apollo, although it is the most popular way developers integrate with Apollo Client. Even if you use another view layer like Angular or Vue, you will still be able to pick up on the concepts covered in the client section and apply them to your view layer of choice.
+As mentioned, we want this example to resemble a real-world Apollo app, so we'll
+also add common useful features like authentication, pagination, and state
+management.
 
-### System requirements
+## Prerequisites
 
-Before we begin, make sure you have:
+This tutorial assumes that you're familiar with both JavaScript/ES6
+and React. If you need to brush up on React, we recommend going through the [official tutorial](https://reactjs.org/tutorial/tutorial.html).
 
-- [Node.js](https://nodejs.org/) v8.x or greater
-- [npm](https://www.npmjs.com/) v6.x or greater
-- [git](https://git-scm.com/) v2.14.1 or greater
+> Building your frontend with React is not a requirement for using the Apollo 
+> platform, but it is the primary view layer supported by Apollo.
+> If you use another view layer (such as Angular or Vue), you can still 
+> apply this tutorial's concepts to it.
+
+### System requirements
 
-While it's not a requirement, we recommend using [VSCode](https://code.visualstudio.com/) as your editor so you can take advantage of all the awesome features the Apollo VSCode extension enables. We're hoping to support other editors in the future.
+Before we begin, make sure you have the following installed:
 
-## Set up your development environment
+- [Node.js](https://nodejs.org/) v8.x or later
+- [npm](https://www.npmjs.com/) v6.x or later
+- [git](https://git-scm.com/) v2.14.1 or later
 
-Now the fun begins! First, you'll need to install our developer tools:
+Although it isn't required, we also recommend using [VS Code](https://code.visualstudio.com/)
+as your editor so you can use Apollo's helpful VS Code extension.
 
-- [Apollo Graph Manager (required)](https://engine.apollographql.com) : Our cloud service where you'll register and manage your graph API.
```

Comment:
Are we moving these suggestions to their relevant sections?

Reply:
Yes, I will do so in subsequent PRs against the appropriate tutorial steps. I think it's valuable to remove them from Step 0 for the time being regardless, because it reduces the perceived amount of pre-work to jumping in.